### ISSUE TITLE
Textwidget mockup v2

### ISSUE THREAD

**Issue by KraXen72** [OxygenCobalt/Auxio#189]
Sorry for opening this Pandora's box again, but with the new widget buttons, i think a *textwidget* would look better than ever 👀👀👀👀👀👀:  
before:  
![Screenshot_20220716-203100_Niagara_Launcher](https://user-images.githubusercontent.com/21956756/179368323-76dfecff-164d-4123-aa54-ca4e8c833e5a.png)  
after:  
![widgets2](https://user-images.githubusercontent.com/21956756/179368327-e54c70bf-d5ab-404c-b860-6cdca48e25ff.png)
  
> might not be possible techically yada yada it's fine to deny this request but damn wouldn't a textwidget look good tho
#### Describe the feature you want to implement:
Add a textwidget

#### Is your feature request related to a problem? Please describe:
Small widget doesen't show song title or album

#### Why do you think this will improve everyone's usage of Auxio?
More useful info on small widgets

#### Due Diligence:
- [x] I have read the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).
- [x] I have read the [Accepted Additions and Requests](https://github.com/OxygenCobalt/Auxio/blob/dev/info/ADDITIONS.md) document.
- [x] I have checked for this feature in the [FAQ](https://github.com/OxygenCobalt/Auxio/blob/dev/info/FAQ.md).

**Comment by OxygenCobalt** [comment]
Actually, given the changes I've made to the playback view in-app, this may be more feasible than I thought. Depends on how I could cram the song information in.

**Comment by OxygenCobalt** [comment]
Okay, so, I can implement it:

![image](https://user-images.githubusercontent.com/65370175/179405838-acbf34e9-14c4-4268-979a-080a902f148f.png)

However, I still don't see myself adding this. I fear the precedent of adding alternative widgets would result in the same scope creep that happens to other apps that eventually results in 6-7 widgets that all are somewhat broken and unusable. I still want to focus effort into a single unified widget. Very sorry about this.

**Comment by KraXen72** [comment]
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
Thank you SO MUCH.

can you maybe send a screenshot where the widget is squashed a bit more? i wanna see what has to go when it's a bit smaller.

**Comment by KraXen72** [comment]
damn was getting excited. You still could probably just first show this instead of the album cover tho?
like the way the widget get's bigger is now: tiny (one button) => medium (squashed) => medium full => big (with text and album cover

what if instead of first showing the album cover a tiny bit (medium squashed), you first showed the song title , and only then the album cover?

**Comment by KraXen72** [comment]
like if you look at the first screenshot of this issue, you can see half of an album cover. You could maybe instead of that firstly show song title and after than, if there is more space, start showing the album cover.

but then again i do not really know how bad the widget api is and if this is even techically possible. It just looks like the widget adds information as it gets bigger, so this could work?

**Comment by OxygenCobalt** [comment]
Yeah, in an ideal world where google did the right thing with widgets, I would do this for the wide widget:
![image](https://user-images.githubusercontent.com/65370175/179412707-7351faf9-9b27-4586-9e2a-1dc0524e35fa.png)

But this is android, and I simply cannot get the album cover to scale correctly at all. Fixed dimensions are off the table, I need the album cover to scale 1:1 like it does in-app, but the hack I use to do such right now simply will not cooperate if it does not have access to the whole height/width of the widget.

As for the look of your current widget, it stems back to how I'm aiming this widget to work best on grid launchers, not the vertical launcher that you use.

**Comment by KraXen72** [comment]
just wanted to say, even tho textwidget is not possible, thanks for this new style. after using this setup for a week, i am starting yo really like it. looks sleek and provides all the playback controls i need, + i usually can make out what the song is from the cover or if i need it then notification. all in all this is better than 1 button. here's how i currently have it set up:![Screenshot_20220723-222558_Niagara Launcher.png](https://user-images.githubusercontent.com/21956756/180620108-6a718b6e-1fdf-4f22-9e82-ac25313e07e8.png)  
TL;DR:
You don't have to reply to this just thanks for the new widget styles 😎👍
